sopel/modules/help.py

```diff
@@ -49,7 +49,8 @@ def help(bot, trigger):
             for line in bot.doc[name][0]:
                 msgfun(line)
             if bot.doc[name][1]:
-                msgfun('e.g. ' + bot.doc[name][1])
+                examples = ', '.join(bot.doc[name][1][:-2] + [' or '.join(bot.doc[name][1][-2:])])
```

Comment:
It's been an _age_ since I did that, but it looks like a grammatical list join—if `examples` needs to output 1 value, it's `value`; 2 values is `value1 or value2`; 3+ is `valueX[, valueN], valueY or valueZ`.

I'll add a comment there.